File: digital_twin/day4/parallel_pro/one.py
from multiprocessing import Process
import time

def taskA():
    for i in range(6, 0, -1):
        print(f"A will exit in {i} seconds")
        time.sleep(1)

def taskB():
    for i in range(3, 0, -1):
        print(f"B will exit in {i} seconds")
        time.sleep(1)

def taskC():
    for i in range(9, 0, -1):
        print(f"C will exit in {i} seconds")
        time.sleep(1)

pa = Process(target=taskA)
pb = Process(target=taskB)
pc = Process(target=taskC)

pa.start()
pb.start()
pc.start()

File: slash/linux/day3/threads/one.c
#include<stdio.h>
#include<unistd.h>
void taska()
{
	int i = 0;
	for (i=3; i>0; i--)
	{
		printf("task A will finish in %d secs\n", i);
		sleep(1);
	}
	return;
}

void taskb()
{
	int i = 0;
	for (i=6; i>0; i--)
	{
		printf("task B will finish in %d secs\n", i);
		sleep(1);
	}
	return;
}

void main()
{
	int i = 0;
	for (i=9; i>0; i--)
	{
		printf("main will finish in %d secs\n", i);
		sleep(1);
	}
	taska();
	taskb();
	return;
}
